pandas/io/tests/compression_tests.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+
+"""
```

Comment:
I think naming this ``compression.py`` is better because then its not runable on its own (its only mixins to others). name ``test_*`` if they are runnable.